pkg/asset/manifests/operators.go

```diff
@@ -264,6 +269,11 @@ func (m *Manifests) Load(f asset.FileFetcher) (bool, error) {
 	return true, nil
 }
 
+func redactedInstallConfig(config types.InstallConfig) ([]byte, error) {
+	config.PullSecret = "XXX"
```

Comment:
nit: Can we just set it to the empty string?

Reply:
I wasn't sure if empty string would trigger some skip mechanism. so I purposefully chose to do `XXX`

but if you think empty is fine, done with https://github.com/openshift/installer/compare/e41fed9f996ec1cc65b05d9fb456ee78d7aca271..141cd3ce50caff295e6dcbad6976ee5e2f344846